src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  &__logo {
+    display: block;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    color: $text-main-color;
+    text-decoration: none;
+    text-transform: uppercase;
+    transition: all $transition-time ease;
+  }
+
+  &__link:hover {
+    color: $header-link-color;
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $header-link-color;
+  &::after {
+    content: '';
+    position: absolute;
+    width: 100%;
+    height: 4px;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    border-radius: 8px;
+    background: $header-link-color;
```

Comment:
It's not clear why you are using border-radius for the bottom line. It does not have visible rounded corners. Consider removing this property.